test/upgrade/mzcompose.py

```diff
@@ -112,12 +114,13 @@ def test_upgrade_from_version(
     c.wait_for_materialized("materialized")
 
     temp_dir = f"--temp-dir=/share/tmp/upgrade-from-{from_version}"
+    seed = f"--seed={random.getrandbits(32)}"
```

Comment:
does `consistent_seed` not work?

Reply:
Unfortunately not! That's what I tried first. But `consistent_seed` is consistent for the duration of the entire workflow, whereas we want a seed that's consistent only within one invocation of this helper function.